everware/spawner.py

```diff
@@ -371,6 +373,9 @@ def start(self, image=None):
                 yield self.remove_old_container()
             self.log.info("Starting container from image: %s" % image_name)
             self._add_to_log('Creating container')
+            #can't mount into /notebooks folder - it deletes all git files
+            if self.everware_yml_param.get("directory_volume"):
+                self.volumes = {self.everware_yml_param.get("directory_volume") : "/notebook/data"}
```

Comment:
directory to map data folder to ('/notebook/data') should be defined inside parse_everware_yml